src/components/SharableLink.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+	<span class="querybuilder__sharable-link">
+		{{ $i18n('query-builder-sharable-link-text') }}
+		<Popover
+			:reactToHover="false"
+			position="end"
+		>
```

Comment:
There's a separate ticket to add the Popover component into the QB.
see [here](https://phabricator.wikimedia.org/T275542). 
If the popover is supposed to react with hover (eventually), we might forget about it by then. I'd say we don't need this here now. but should be done in the dedicated ticket. 

I know it says it in the ticket. but there's a ticket dedicated for dealing with popovers in the QB